smart-contracts/testdata/contracts/.diff-wat-wasm-ignore

```diff
@@ -0,0 +1,2 @@
+./mut-global-offset-test.wat
```

Comment:
Why are these ignored? I guess you can't leave a comment here

Reply:
They are ignored because they aren't valid modules and thus can't be compiled. As is also stated inside the wat files.
They are invalid because of how they use globals in constant expressions, which we had a bug related to 1-2 years ago.

I can add comments, but the script will just handle them as another file to ignore :p